scripts/set_gradle_proxy.sh

```diff
@@ -22,8 +22,18 @@ writeProxyStringToGradleProps () {
     PROTOCOL=$2
     FILE=$3
 
+    # Strip the port.
     HOST=${PROXY%:*}
+    # Strip the protocol.
     HOST=${HOST#*//}
+    # Extract authentication info.
+    AUTH=${HOST%%@*}
```

Comment:
I wonder if it'd make sense to declare the variables only relevant to this functions as `local`, even if it does not comply with POSIX but seems to be supported by most shells, [see stackoverflow](https://stackoverflow.com/questions/18597697/posix-compliant-way-to-scope-variables-to-a-function-in-a-shell-script). What do you think?

Reply:
Yeah, good idea, done.

Reply:
Thanks!